api/swagger-spec/v1.json

```diff
@@ -12394,6 +12394,14 @@
      "securityContext": {
       "$ref": "v1.SecurityContext",
       "description": "security options the pod should run with; see http://releases.k8s.io/HEAD/docs/design/security_context.md"
+     },
+     "tty": {
```

Comment:
We'll also need to regenerate API docs for all these API changes.

cc @caesarxuchao